docs/development.md

```diff
@@ -78,7 +81,14 @@ Then run `M-x add-dir-local-variable RET python-mode RET pyvenv-activate RET <pa
 ### Other setups
 
 - Install and enable [ShellCheck](https://github.com/koalaman/shellcheck) to find bash errors locally.
-- If you need to update the documentation, it is recommended to install the extension [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). It helps to maintain the table of contents in the documentation.
+- If there is need to update the documentation, it is recommended to
+  install the extension [Markdown All in
+  One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one).
+  It helps to maintain the table of contents in the documentation.
+- While the black formatter does not handle long comments and the like
+  well (https://github.com/psf/black/issues/1331), you might want to
```

Comment:
@glima I will merge those PRs. I may update the statements later. We represent Microsoft's points. we can provide constructive suggestions what we can. And we consider what value provided to users.